src/terrama2/impl/DataAccessorGrADS.cpp

```diff
@@ -279,7 +263,29 @@ terrama2::core::DataSetSeries terrama2::core::DataAccessorGrADS::getSeries(const
     datasetMask = replaceMask(datasetMask.c_str()).toStdString();
 
 
-    for (const auto& dataFileInfo : newFileInfoList)
+    QFileInfoList binFileList;
```

Comment:
It's inside the loop for the CTL, so this list contains only files that are valid for the dataset name from the current CTL file